gui/src/containers/JobSpecRuns.js

```diff
@@ -23,47 +25,54 @@ const styles = theme => ({
   title: {
     marginTop: theme.spacing.unit * 5,
     marginBottom: theme.spacing.unit * 5
+  },
+  customButtons: {
+    flexShrink: 0,
+    color: theme.palette.text.secondary,
+    marginLeft: theme.spacing.unit * 2.5,
   }
 })
 
-const START_PAGE = 1
-
 export class JobSpecRuns extends Component {
-  constructor (props) {
+  constructor(props) {
     super(props)
     this.state = {
       page: 0
     }
     this.handleChangePage = this.handleChangePage.bind(this)
   }
 
-  componentDidMount () {
-    const {jobSpecId, pageSize, fetchJobSpecRuns} = this.props
-    fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+  componentDidMount() {
+    const { jobSpecId, pageSize, fetchJobSpecRuns } = this.props
+    if (this.props.match.params.jobRunsPage) {
+      const START_PAGE = this.props.match.params.jobRunsPage
+      this.setState({ page: START_PAGE - 1 })
+      fetchJobSpecRuns(jobSpecId, START_PAGE, pageSize)
+    } else {
+      this.setState({ page: 0})
+      fetchJobSpecRuns(jobSpecId, 1, pageSize)
```

Comment:
Why the off by one discrepancy? Rather than `setState` be 0 based and `fetchJobSpecRuns` be 1 based, let's make it all 1 based.

Reply:
From what I remember from last night, the table tabs were needed to be 0 based.